prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -1,6 +1,7 @@
 <app-bcsc-profile [user]="enrollee"></app-bcsc-profile>
 
-<app-page-section *ngIf="identityProvider === IdentityProviderEnum.BCSC || identityProvider === IdentityProviderEnum.IDIR">
+<app-page-section
```

Comment:
Could you please explain? This change happened after I Ctrl + S, which put the *ngIf one line under?

Reply:
Could you please explain? This change happened after I Ctrl + S, which put the *ngIf one line under

Reply:
We should look at your settings in VSCode since that doesn't happen for myself or appear to happen in other PRs.  The first attribute should align with the element and then stack, but that should be automagic.

Reply:
I wonder if this is due to line length.  What happens if you reduce the line length does it line up?  If it does just leave it the way it is.

Reply:
It is due to the line length